### DISCUSSION TITLE
Ability to set keybinds for mod presets

### DISCUSSION THREAD

**Discussion by Loreos7** [ppy/osu#30629] [Ideas]
Find it pretty useful. Every user's mod preset must be shown up in keybinds listing with `none` setting by default.
Users can use these bindings in song select menu only.

![Untitled-1](https://github.com/user-attachments/assets/9e4e86ce-b924-49c7-8e0c-4b6deb912112)

Related: https://github.com/ppy/osu/discussions/19683

**Comment by cl8n** [comment]
I prefer this over the other linked suggestion. would be a huge QoL improvement in the same way that ctrl+up/down was for rates

**Comment by peppy** [comment]
Before adding this, I think I'd want to see the game remember the previous mods on startup.
